checkov/cloudformation/cfn_utils.py

```diff
@@ -126,6 +127,19 @@ def build_definitions_context(definitions, definitions_raw, root_folder):
                     if isinstance(attr_value, dict_node):
                         start_line = attr_value.start_mark.line
                         end_line = attr_value.end_mark.line
+                        # fix lines number for yaml and json files
+                        if file_path.endswith(".yaml") or file_path.endswith(".yml"):
```

Comment:
this can be rewritten as:
`if file_ending in CF_POSSIBLE_ENDINGS:`

Reply:
but I want only files in yaml format